pure_adaptor/processor.py

```
@@ -24,11 +24,20 @@ def __init__(self,
                  instance_id,
                  input_stream,
                  invalid_stream,
-                 error_stream):
+                 error_stream,
+                 pure_flow_limit):
 
         self.instance_id = instance_id
         self.api_version = api_version
         self.pure = versioned_pure_interface(api_version)
+        flow_limit = 60
+        try:
+            flow_limit = int(pure_flow_limit)
+        except ValueError:
+            # Handle the exception
+            logging.exception('Flow Limit not an integer, defaulting to 60')
```

If we get here, would it be better to just let the exception bubble up and show in the logs + the fact we won't make any requests, since it means the service has been misconfigured? Better to find out at deploy time + initial QA, rather than somewhere down the line?

---

change made